client/src/components/QandA/QandA.jsx

```diff
@@ -9,6 +12,7 @@ class QnA extends React.Component {
       questionSearch: ''
     }
     this.changeQuestion = this.changeQuestion.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
   }
 
   changeQuestion(event) {
```

Comment:
Is this missing a closing bracket? Or am I just not seeing it?

Reply:
No! These are part of of the the constructor.